lib/modules/solidity/solcP.js

```diff
@@ -1,41 +1,58 @@
-let solc;
-
 const fs = require('fs-extra');
 const path = require('path');
 const constants = require('../../constants');
 const Utils = require('../../utils/utils');
 
-function findImports(filename) {
-  if (filename.startsWith('http') || filename.startsWith('git')) {
-    const fileObj = Utils.getExternalContractUrl(filename);
-    filename = fileObj.filePath;
-  }
-  if (fs.existsSync(filename)) {
-    return {contents: fs.readFileSync(filename).toString()};
+const ProcessWrapper = require('../../process/processWrapper');
+
+class SolcProcess extends ProcessWrapper {
+
+  findImports(filename) {
```

Comment:
👍

Reply:
perhaps lay down some comments on *how* this finds imports and emphasize that it can take a git url/http path and all the various ways that it can be used. Because this is probably going to be a highly used process.